english/lego-world/lego-world.md

```diff
@@ -0,0 +1 @@
+You step through the enchanted door and into Lego World! But this world is a Lego version of Willy Wonka and the Chocolate Factory. You are actually skiing down the slope of a marshmallow mountain made out of Legos! You are surrounded by lollipop trees and gingerbread houses. Everything is awesome!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.